src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateMethodTests.cs

```diff
@@ -7246,7 +7246,7 @@ public void M1()
 parseOptions: TestOptions.Regular);
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/15508"), Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
```

Comment:
My mistake. The parser does not do that. The parser produces `(int, int)` as an expression, and the IDE code chokes on that. I'll fix the bug description.

---
In reply to: [90171673](https://github.com/dotnet/roslyn/pull/15548#discussion_r90171673) [](ancestors = 90171673)